src/Elm.elm

```diff
@@ -10,8 +10,10 @@ module Elm exposing
     , Declaration
     , comment, declaration
     , withDocumentation
-    , expose, exposeWith
+    , group
```

Comment:
I think a `withGroup : Declaration -> Declaration` could also make sense?

Reply:
I don't think it would be needed, and one can keep using `exposeWith` in that case?

Thinking more about it, I think that pushing towards having a group close to each other is good